cleanlab/count.py

```diff
@@ -576,6 +576,7 @@ def compute_confident_joint(
 
     # labels needs to be a numpy array
     labels = np.asarray(labels)
+    # replacing with labels = labels_to_array(labels) resolves unit-test
```

Comment:
We should not do this as we intentionally want to preserve the nested array format in `labels` for some of the following function calls (eg. `get_confident_thresholds`).

Instead you can call `labels_to_array(labels)` in `classification.find_label_issues` when intended